pkg/apis/autoscaling/annotation_validation.go

```diff
@@ -31,8 +31,8 @@ func getIntGE0(m map[string]string, k string) (int64, *apis.FieldError) {
 	i, err := strconv.ParseInt(v, 10, 32)
 	if err != nil || i < 0 {
 		return 0, &apis.FieldError{
```

Comment:
We have a pre-defined Invalid value type that may be nicer to user here.: https://github.com/knative/pkg/blob/master/apis/field_error.go#L322

If we want to give range information we also have a ErrorOutOfBoundsValue: https://github.com/knative/pkg/blob/master/apis/field_error.go#L359